curvefs/src/client/s3/client_s3_cache_manager.h

```diff
@@ -311,6 +317,8 @@ class FsCacheManager {
     }
 
     bool WriteCacheIsFull() {
+        if (writeCacheMaxByte_ <= 0)
```

Comment:
under what condition write cache <=0?

Reply:
writeCacheMaxByte_  is a configuration item, just prevent divisor from being 0